hyperspy/learn/rpca.py

```diff
@@ -128,7 +126,7 @@ def svd(X):
         Y2 = np.random.randn(n, rank)
         for i in range(power + 1):
             Y2 = np.dot(L.T, np.dot(L, Y2))
-        Q, tmp = scipy.linalg.qr(Y2, mode='economic')
+        Q, tmp = scipy.linalg.qr(Y2, mode="economic")
```

Comment:
Were these changes from ' to " also intentional? I'd always preferred ' so curious as to why if so?

Reply:
This is thanks to [black](https://black.readthedocs.io/en/stable/index.html)